readme/Index.scalatex

```diff
@@ -368,6 +368,29 @@
     @p
       The @hl.scala{debug()} call returns @hl.scala{: Any}; by default, this is @hl.scala{(): Unit}, but you can also return custom values by passing in an argument to @hl.scala{exit(...)} when you exit the REPL. This value will then be returned from @hl.scala{debug()}, and can be used in the rest of your Scala application.
 
+    @p
+      Ammonite can also be used to remotely connect to your running application and interact with it in real-time, similar to erlang `erl -remsh` command. To do this, add ammonite sshd server to your classpath, for example with SBT:
+
+    @hl.scala
+      libraryDependencies += "com.lihaoyi" % "ammonite-repl-sshd" % "@ammonite.Constants.version" cross CrossVersion.full
+
+    @p
+      Now add repl server to your application:
+
+    @hl.scala
+      import ammonite.repl.sshd._
+      val replServer = new SshdRepl(SshServerConfig(port = 22222, username = "repl", password = "your_secure_password"))
```

Comment:
It doesn't supports key-based auth currently. I've seen it as a too much work for a beginning, but of course it could be added later.

SSH always asks for password in case there is no key-based auth, but you are free to left password blank if you wish. There is no other way to make passwordless auth other than enabling key verification AFAIK.

WRT port if you have it opened everybody will be able to access it. My personal recommendation would be to use non-standard port for a couple of reasons:
* port 22 most likely already used by system's sshd
* port 22 may be reserved to be opened only with root access
* it's not a very usual sshd shell access so might be a little unexpected to be found at port 22
* very basic protection from potential attacks - if nobody know which port to attack there are less chances it will be attacked